api/filesystem/filesystem.cpp

```diff
@@ -27,6 +30,13 @@ using namespace std;
 using json = nlohmann::json;
 
 namespace fs {
+    
+#if defined(_WIN32)
+// From: https://stackoverflow.com/a/6161842/3565513
```

Comment:
Hey @pathange-s  thank you very much for sending this pull request. Could you please let me know if you are okay with adding this comment to your helper function implementation? We always love to give credit to people who provide these free solutions. If you found this solution from another source (or by yourself) Please let me know too ;) I think after discussing this we can merge this pull request :tada:

Reply:
Yes! I am okay with the following comment. Infact, loved the idea of mentioning it. Also, I tweaked it to make it suitable for our need.

Reply:
Great! :tada:

Reply:
Hmm.. I was thinking a bit about `createdAt` since it's not really the creation time on Linux (rather an inode change timestamp). However, let's merge this one -- we can think about it before the next release ;)

Reply:
https://github.com/neutralinojs/neutralinojs.github.io/pull/204/files#diff-987ec3372f278ebc4449566c57aea73655f1a17e1d8840e875ae0a9ec70cad52R186
"It depends on the Operating system"